### PR TITLE
New functional tests

### PR DESCRIPTION
**Pull request by mhagen-vmware**
Some new functional tests, which required a change to Util.robot in order to support installing into a VC with multiple DCs

### CODE REVIEW

File: tests/manual-test-cases/Group5-Functional-Tests/5-10-Multiple-Datacenter.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+=======
+
+#Purpose:
+To verify the VIC appliance works when the vCenter appliance has multiple datacenters
+
+#References:
+[1 - VMware vCenter Server Availability Guide](http://www.vmware.com/files/pdf/techpaper/vmware-vcenter-server-availability-guide.pdf)
+
+#Environment:
+This test requires access to VMWare Nimbus cluster for dynamic ESXi and vCenter creation
```
Comment (anchal-agrawal) [inline tests/manual-test-cases/Group5-Functional-Tests/5-10-Multiple-Datacenter.md]: `VMware` :)

File: tests/manual-test-cases/Group5-Functional-Tests/5-11-Multiple-Cluster.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+=======
+
+#Purpose:
+To verify the VIC appliance works when the vCenter appliance has multiple clusters within the datacenter
+
+#References:
+[1 - VMware vCenter Server Availability Guide](http://www.vmware.com/files/pdf/techpaper/vmware-vcenter-server-availability-guide.pdf)
+
+#Environment:
+This test requires access to VMWare Nimbus cluster for dynamic ESXi and vCenter creation
```
Comment (anchal-agrawal) [inline tests/manual-test-cases/Group5-Functional-Tests/5-11-Multiple-Cluster.md]: ditto

**Comment by anchal-agrawal** [comment]
LGTM

File: tests/manual-test-cases/Group5-Functional-Tests/5-10-Multiple-Datacenter.md (hunk 1 above)
Comment (matthewavery) [inline tests/manual-test-cases/Group5-Functional-Tests/5-10-Multiple-Datacenter.md]: Not here too!!!

**Comment by matthewavery** [comment]
LGTM

**Comment by mhagen-vmware** [comment]
.

**Pull request merged**